server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -116,14 +142,20 @@ def mock_results_push(self, ctrl, tb, token):
 
     def test_process_tb_zero(self, monkeypatch):
         """verify processing if there are no TBs without any failure"""
+        tb_path = fixture_path / "tarball"
+
+        def mock_glob(*args, **kwargs):
+            return []
```

Comment:
Because the glob returns an empty list, `results_push()` should _not_ be called, so it would be good to have a mock for it which fails the test if it does get called.